src/util/fetch.js

```diff
@@ -1,15 +1,15 @@
 export const fetchJson = (url, options = {}) => {
-    const requestHeaders = new Headers({
+    const requestHeaders = options.headers || new Headers({
         Accept: 'application/json',
     });
     if (!(options && options.body && options.body instanceof FormData)) {
         requestHeaders.set('Content-Type', 'application/json');
     }
-    if (options.user && options.user.authenticated && options.user.authenticated) {
+    if (options.user && options.user.authenticated && options.user.token) {
         requestHeaders.set('Authorization', options.user.token);
     }
 
-    return fetch(url, { ...options, headers: requestHeaders, credentials: 'include' })
```

Comment:
I shouldn't say that, but there are other small BC breaks since 0.4.0 that aren't mentioned anywhere. Let's consider that we're still in alpha and have it shipped fast instead ;)